data-es6.js

```diff
@@ -2914,6 +2914,30 @@ exports.tests = [
   },
 },
 {
+  name: 'prototype of bound functions',
+  category: 'functions',
+  significance: 'small',
+  link: 'https://people.mozilla.org/~jorendorff/es6-draft.html#sec-boundfunctioncreate',
+  exec: function () {/*
+    function getProtoBound(proto) {
+      var f = Object.setPrototypeOf(function() { }, proto)
```

Comment:
Right, but these tests need to run in older engines that have shimmed behavior. Since `Function#bind` is shimmable, the `es6-shim` will eventually correct this behavior, and would need to be able to pass in IE 9. Otherwise the test will only pass in browsers, never with shims/transpilers.